mne/viz/topomap.py

```diff
@@ -145,13 +145,37 @@ def _add_colorbar(ax, im, cmap, side="right", pad=.05, title=None,
     divider = make_axes_locatable(ax)
     cax = divider.append_axes(side, size=size, pad=pad)
     cbar = plt.colorbar(im, cax=cax, cmap=cmap, format=format)
-    if cmap[1]:
+    if cmap is not None and cmap[1]:
         ax.CB = DraggableColorbar(cbar, im)
     if title is not None:
         cax.set_title(title, y=1.05, fontsize=10)
     return cbar, cax
 
 
+def _remove_zeros(proj):
+    """Remove grad or mag data if only contains 0s (gh 5641)."""
+    import re
+    grad_regex = '^MEG.*[23]$'
+    meg_regex = '^MEG.*1$'
+
+    proj = copy.deepcopy(proj)
+    proj['data']['data'] = np.atleast_2d(proj['data']['data'])
```

Comment:
I would add something like `assert proj['data']['data'].ndim == 2 and proj['data']['data'].shape[0] == 1` or so and then do `pytest mne -k proj` and see where the assertion fails. Presumably it will somewhere

Reply:
yes. That's what I did, but this only traces whatever prints the topo maps. Which in that case, using `np.atleast_2d` is good enough. (the only breaking test was `test_plot_topomap_neuromag122` which is how I found about it)

My question was more, where would you put the asserts like those you proposed in the code such that things break when running the entire testsuit + doc